plugins/generator-1.14.4/forge-1.14.4/procedures/block_fluid_to_bucket.java.ftl

```diff
@@ -0,0 +1,3 @@
+<#include "mcitems.ftl">

```

Comment:
Please add the "/\*@ItemStack\*/" tag at the beginning of this file as well as its 1.15.2 counterpart.